package.json

```diff
@@ -16,22 +16,24 @@
   ],
   "license": "Apache-2.0",
   "dependencies": {
-    "babel-plugin-syntax-trailing-function-commas": "^6.22.0",
-    "babel-plugin-transform-es2015-modules-commonjs": "^6.26.2",
-    "babel-plugin-transform-es2015-template-literals": "^6.22.0",
-    "babel-plugin-transform-es3-member-expression-literals": "^6.22.0",
-    "babel-plugin-transform-es3-property-literals": "^6.22.0",
-    "babel-plugin-transform-es5-property-mutators": "^6.24.1",
-    "babel-plugin-transform-exponentiation-operator": "^6.24.1",
-    "babel-plugin-transform-jscript": "^6.22.0",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
+    "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
+    "@babel/plugin-transform-exponentiation-operator": "^7.0.0",
+    "@babel/plugin-transform-jscript": "^7.0.0",
+    "@babel/plugin-transform-member-expression-literals": "^7.0.0",
+    "@babel/plugin-transform-modules-commonjs": "^7.0.0",
+    "@babel/plugin-transform-property-literals": "^7.0.0",
+    "@babel/plugin-transform-property-mutators": "^7.0.0",
+    "@babel/plugin-transform-template-literals": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
+    "@babel/preset-react": "^7.0.0",
     "babel-plugin-transform-react-remove-prop-types": "^0.4.15",
-    "babel-plugin-transform-strict-mode": "^6.24.1",
-    "babel-preset-env": "^1.7.0",
-    "babel-preset-react": "^6.24.1",
     "object.assign": "^4.1.0"
   },
+  "peerDependencies": {
+    "@babel/core": "^7.0.0-0"
```

Comment:
should this be "7.0.0-0" and not just "^7.0.0"?

Reply:
I remember reading that this is the correct peerDep.

If you take a look [at any plugin made by babel, the peerDependency is the same](
https://github.com/babel/babel/blob/master/packages/babel-plugin-proposal-class-properties/package.json#L23).

Reply:
It has to do with prerelease versions; the `-0` is required to *allow* them.

Reply:
@ljharb I'm on the fence here. Since all the plugins are stable they may not be compatible with a random beta anyway, I'm not sure you gain anything from this now that core is stable and this will be a new major for the preset anyway.

Reply:
Fair enough; I'm also fine with starting with "latest"